packages/dx-vue-grid/docs/reference/table-group-row.md

```diff
@@ -0,0 +1,141 @@
+# DxTableGroupRow Plugin Reference
+
+A plugin that renders group rows and enables them to expand and collapse.
+
+## Importing
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { DxTableGroupRow } from '@devexpress/dx-vue-grid-bootstrap4';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { DxTableGroupRow } from '@devexpress/dx-vue-grid';
+```
+
+## User Reference
+
+### Dependencies
+
+- [DxGroupingState](grouping-state.md)
+- [DxTable](table.md)
+- [DxDataTypeProvider](data-type-provider.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+showColumnsWhenGrouped? | boolean | false | A Boolean value that specifies whether the grid's table displays a column by which data is grouped.
+columnExtensions? | Array&lt;[DxTableGroupRow.ColumnExtension](#dxtablegrouprowcolumnextension)&gt; | | Additional column properties that the plugin can handle.
+cellComponent | object | [DxTableGroupRow.components.DxCell](#dxtablegrouprowcomponentsdxcell) | A component that renders a group cell.
+rowComponent | object | [DxTableGroupRow.components.DxRow](#dxtablegrouprowcomponentsdxrow) | A component that renders a group row.
+indentCellComponent? | [DxTableGroupRow.IndentCellProps](#dxtablegrouprowindentcellprops) | null | A component that renders a group indent cell.
+indentColumnWidth | number | | The group indent column's width.
+
+## Interfaces
+
+### DxTableGroupRow.ColumnExtension
+
+Describes additional column properties that the plugin can handle.
+
+Field | Type | Description
+------|------|------------
+columnName | string | The name of a column to extend.
+showWhenGrouped? | boolean | Specifies whether the grid displays the column by which data is grouped.
+
+### GroupRow
+
+Describes the group row structure.
+
+Field | Type | Description
+------|------|------------
+key | number &#124; string | The current group key.
+value | any | The current group value.
+
+### DxTableGroupRow.IndentCellProps
```

Comment:
It should be converted to the component

Reply:
We [don't export](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-vue-grid-bootstrap4/src/plugins/table-group-row.js#L18) this component. Well, I decided to describe it as a type.